examples/LossMinimization/loss_minimization.jl

```diff
@@ -1,5 +1,8 @@
 # # Minimization of simple loss functions
 #
+# !!! info "How do I run this code?"
```

Comment:
uncomment? Currently "The full code is found in ..." is outside of the "How do I run this code?" box

Reply:
It's actually an indenting error i think with the !!! info block, resolving...